### PR TITLE
feat: add link from xblock staff area to ESG

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** add link from xblock staff area to ESG

JIRA: [AU-569](https://openedx.atlassian.net/browse/AU-569)

**What Changed?**

- Added a link to ESG demo when ESG is not enabled
- Made the staff area buttons inline-block with a margin because they are now overflowing
- I tried to mimic a Paragon Button in terms of coloring: used primary-500 for background, darkened by 7.5% on hover

### New Button
primary-500
### Hover
primary-500-hover
### Close comparison of hover vs non-hover
focus-cmp

- **Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/static/sass/oa/utilities/_developer.scss [hunk 1]
```diff
@@ -72,6 +73,16 @@
         background-color: $action-primary-color;
       }
     }
+
+    .button-enhanced-staff-grader-demo {
+      color: white;
+      background-color: $action-primary-color;
+
+      // &:hover {
```
Comment (jansenk) [inline openassessment/xblock/static/sass/oa/utilities/_developer.scss:81]: This should be deleted

**Review by leangseu-edx — CHANGES_REQUESTED** [review]
`make static` isn't necessary here.

**Comment by jansenk** [comment]
> `make static` isn't necessary here.

Don't I need it for my css changes?

**Comment by leangseu-edx** [comment]
Nvm, you are right. `make static` is necessary for the css change.

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Comment by jansenk** [comment]
I'm planning to just ignore coverage.

File: openassessment/data.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def _use_read_replica(queryset):
     )
 
 
-def _get_course_blocks(course_id):
```
[the remark this answers is not in the document]
Comment (jansenk) [reply openassessment/data.py:57]: codecov was complaining that total project coverage was too low, so I no-covered this block (as it's expected to not be covered, since it's one of those runtime imports) so that it wouldn't complain. I then cherry-picked it onto master https://github.com/openedx/edx-ora2/pull/1843

File: openassessment/xblock/static/js/spec/lms/oa_staff_area.js [hunk 3]
```diff
@@ -354,12 +354,13 @@ describe('OpenAssessment.StaffAreaView', function() {
...
                 $buttons = $('.ui-staff__button', view.element);
-            expect($buttons.length).toBe(4);
+            expect($buttons.length).toBe(5);
             expect($buttons).toHaveAttr('aria-expanded', 'false');
             expect($($buttons[0]).text().trim()).toEqual('Manage Individual Learners');
             expect($($buttons[1]).text().trim()).toEqual('View Assignment Statistics');
             expect($($buttons[2]).text().trim()).toEqual('Grade Available Responses');
-            expect($($buttons[3]).text().trim()).toEqual('View ORA in Studio');
+            expect($($buttons[3]).text().trim()).toEqual('Demo the new Grading Experience');
+            expect($($buttons[4]).text().trim()).toEqual('View ORA in Studio');
```
Comment (nsprenkle) [inline openassessment/xblock/static/js/spec/lms/oa_staff_area.js:363]: Are we missing the negative version of this test? When ESG is enabled, should not show the Demo/Grade buttons.

**Review by nsprenkle — APPROVED** [review]
